logistics_planning_sale/models/sale_order_line.py

```diff
@@ -39,6 +39,14 @@ class SaleOrderLine(models.Model):
         are allowed for this line
         """,
     )
+    logistics_vehicle_type_id = fields.Many2one(
```

Comment:
This field is uneeded, as `stock_picking_mgmt_weight` already adds it (see https://github.com/solvosci/slv-stock/blob/8b8dea9b8bb6ac0648c33eda7d989aec20b0c3de/stock_picking_mgmt_weight/models/sale_order_line.py#L36)